src/streaming/TextTracks.js

```diff
@@ -423,17 +426,11 @@ function TextTracks() {
                 };
             }
             else if (currentItem.type === 'html') {
+                log('Create Cue id:' + currentItem.cueID + ' start : ' + currentItem.start + ' end : ' + currentItem.end + ' timeOffset : ' + timeOffset);
                 cue = new Cue(currentItem.start - timeOffset, currentItem.end - timeOffset, '');
                 cue.cueHTMLElement = currentItem.cueHTMLElement;
-                cue.regions = currentItem.regions;
-                cue.regionID = currentItem.regionID;
+                cue.isd = currentItem.isd;
                 cue.cueID = currentItem.cueID;
-                cue.videoWidth = currentItem.videoWidth;
-                cue.videoHeight = currentItem.videoHeight;
-                cue.cellResolution = currentItem.cellResolution;
```

Comment:
I thought that too but Tobbe implied scaling was already working in a comment on the other PR, so that's why I thought scaleCue might not be needed.